src/y0/examples.py

```diff
@@ -138,6 +142,167 @@ class Example:
     graph=m_graph,
 )
 
+# NxMixedGraph containing vertices without edges
+vertices_without_edges = Example(
+    name="Vertices-without-Edges",
+    reference="out of the mind of JZ (patent pending). See NFT for details",
+    graph=NxMixedGraph.from_adj(
+        directed={"W": [], "X": ["Y"], "Y": ["Z"], "Z": []},
+        undirected={"W": [], "X": ["Z"], "Y": [], "Z": []},
+    ),
+)
+
+# Line 1 example
+line_1_example = Example(
+    name="Line 1 of ID algorithm",
+    reference="out of the mind of JZ",
+    graph=NxMixedGraph.from_edges(
+        directed=[
+            ("Z", "Y"),
+        ]
+    ),
+    identifications=[
+        dict(
+            id_in=[
```

Comment:
is this dict always two keys? If that's the case, I'd suggest a named tuple data structure instead